pkg/api/utils/serviceUtils.go

```diff
@@ -78,15 +79,6 @@ func (s *ServiceHandler) getHTTPClient() *http.Client {
 	return s.HTTPClient
 }
 
-// CreateService creates a new service
```

Comment:
Just want to double-check: this method is obsolete?

Reply:
Yes, this function has been moved into the apiUtils.